functions/src/express_configs/express_secure.ts

```diff
@@ -8,24 +8,48 @@ import express from "express";
 import jwt from "express-jwt";
 import jwksRsa from "jwks-rsa";
 import * as Sentry from "@sentry/node";
+import * as Tracing from "@sentry/tracing";
 import cors from "cors";
 import * as bodyParser from "body-parser";
 import { Response, Request } from "express";
 
 const app = express();
 
 //setup sentry
-if (functions.config()?.sentry?.dns) Sentry.init({ dsn: functions.config().sentry.dns });
+if (functions.config()?.sentry?.dns) {
+  Sentry.init({
+    dsn: functions.config().sentry.dns,
+    integrations: [
+      // enable HTTP calls tracing
+      new Sentry.Integrations.Http({ tracing: true }),
+      // enable Express.js middleware tracing
+      new Tracing.Integrations.Express({ app }),
+    ],
+    tracesSampleRate: 1.0,
+  });
+}
 
 // The request handler must be the first middleware on the app
 app.use(Sentry.Handlers.requestHandler());
+// TracingHandler creates a trace for every incoming request
+app.use(Sentry.Handlers.tracingHandler());
 
 app.use(cors({ origin: true }));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 // The error handler must be before any other error middleware and after all controllers
-app.use(Sentry.Handlers.errorHandler());
+app.use(
+  Sentry.Handlers.errorHandler({
+    shouldHandleError(error) {
+      // Capture all errors over 400
+      if ((error.status as number) >= 400) {
+        return true;
+      }
+      return false;
+    },
```

Comment:
Is there a reason we can't simplify the code to this:
```suggestion
    shouldHandleError: (error) => (error.status as number) >= 400,
```